### ISSUE TITLE
Unable to acquire a buffer item, very likely client tried to acquire more than maxImages buffers

### ISSUE THREAD

**Issue by diybl** [juicycleff/flutter-unity-view-widget#308]
Stack(
          children: [
            Container(
              child: UnityWidget(
                isARScene: true,
                safeMode: true,
                fullscreen: true,
                onUnityCreated: onUnityCreated,
                onUnityMessage: onUnityMessage,
                onUnitySceneLoaded: onUnitySceneLoaded,
              ),
            ),
            Positioned(
              left: 30,
              bottom: 30,
              child: JoystickView(
                  size: 150,
                  backgroundColor: Color(0x00000000),
                  circleBorderColor: Color(0x10FFFFFF),
                  innerCircleColor: Color(0x00000000),
                  innerCircleBorderColor: Color(0x20FFFFFF),
                  iconsColor:Color(0x20000000),
                  showArrows: false
              ),
            ),
            Positioned(
              right: 30,
              bottom: 30,
              child: CircleView(
                size: 40,
                color: Color(0x20000000),
              ),
            ),
          ],
        ),

**Comment by chenenyu** [comment]
Me too, i rolled back to virtual display mode.

**Comment by cmd-dev** [comment]
> Me too, i rolled back to virtual display mode.

How did you do that? Did it fix the issue? @chenenyu

**Comment by chenenyu** [comment]
@cmd-dev Yes. Just see this commit https://github.com/juicycleff/flutter-unity-view-widget/commit/b5dd0d903ee538e237ab068622f92259311b9a7a
